packages/runtime/runtime-utils/src/summarizerNode/summarizerNodeWithGc.ts

```diff
@@ -142,8 +142,8 @@ class SummarizerNodeWithGC extends SummarizerNode implements IRootSummarizerNode
 		});
 
 		this.childNodesBaseGCDetailsP = new LazyPromise(async () => {
-			const baseGCDetails = await this.baseGCDetailsP;
-			return unpackChildNodesGCDetails(baseGCDetails);
+			await this.loadBaseGCDetails();
```

Comment:
Ok this took me a minute to understand.  Let me see if I got it right.  This change matters for the DataStore - When the DDS is realized sometime after Refresh happens, this LazyPromise will be called from `createChild`, and the call to `this.loadBaseGCDetails` will be a no-op, since it was called already during refresh (or earlier). Furthermore, `this.gcData` was modified during refresh.

And in the case where `this.loadBaseGCDetails` has not been called yet, this code behaves the same as before.